### ISSUE TITLE
Why it doesn't have the first step to create the project directory?

### ISSUE THREAD

**Issue by enBonnet** [zach-hopkins/create-t3svelte-app#21]
It feels, to me, that we are missing the first step. Where the CLI ask the name of the project to create the directory and set that name everywhere.

There a reason why this package skips it?
Do you want help to add it?

**Comment by zach-hopkins** [comment]
Yes this is something i've been meaning to add! I plan on getting to it shortly. I certainly wouldn't object to a PR which adds the functionality.
